<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectServer</class>
 <widget class="QMainWindow" name="ConnectServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TesTrack </string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 175, 110, 255));
}
#frame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 217, 0, 255), stop:1 rgba(255, 165, 92, 255));
border-radius: 20px;
}
QLineEdit {
    font: 8pt MS Shell Dlg 2;
    background-color: rgb(255, 255, 255);
    border-radius: 18px;
    padding-left: 10px;
    padding-right: 10px;
}
QPushButton {
    background-color: rgb(255, 255, 255);
    color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 217, 0, 255), stop:1 rgba(255, 165, 92, 255));;
    padding: 6px;
    margin-top: 10px;
    border-radius: 10px;
}
QLineEdit:focus{
    background-color: rgb(221, 221, 221);
}
QPushButton:hover,
QPushButton:clicked {
    background-color: rgb(221, 221, 221);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Вход</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../CheatHunter/client/lock.png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../CheatHunter/client/user.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdmin_Tools">
    <property name="title">
     <string>Admin Tools</string>
    </property>
    <addaction name="actionAdmin_Panel"/>
    <addaction name="actionServer_Settings"/>
   </widget>
   <addaction name="menuAdmin_Tools"/>
  </widget>
  <action name="actionAdmin_Panel">
   <property name="text">
    <string>Admin Panel</string>
   </property>
  </action>
  <action name="actionServer_Settings">
   <property name="text">
    <string>Server Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
